widget/form_test.go

```diff
@@ -18,7 +17,7 @@ func TestFormSize(t *testing.T) {
 
 func TestForm_CreateRenderer(t *testing.T) {
 	form := &Form{Items: []*FormItem{{Text: "test1", Widget: NewEntry()}}}
-	assert.NotNil(t, test.WidgetRenderer(form))
+	assert.NotNil(t, Renderer(form))
```

Comment:
This is great, thanks. Ready to merge but then I noticed this.
The change goes back to using a deprecated method from the new test one that looks up correctly...

Reply:
Well it's automatic goimports change that used by editor on saving, I don't know why it happens =)
I'll back it by hand.

Reply:
Ah, I got it. I made first changes on master branch and only after that switched to develop branch but forget properly update file. Fixed now.